matter/wifi/wf200/efr_spi.c

```diff
@@ -62,23 +62,35 @@
 #include "sl_power_manager.h"
 #endif
 #include "AppConfig.h"
-
+#if defined(EFR32MG12)
 #define USART SL_WFX_HOST_PINOUT_SPI_PERIPHERAL
+#elif defined(EFR32MG24)
+#define MY_USART SL_WFX_HOST_PINOUT_SPI_PERIPHERAL
```

Comment:
Why are two names used? for MG12 `USART` for mg24 `MY_USART`. It can lead to confusion and also prevent reusable code/ causing more #ifdef everywhere

Reply:
Modified as per the suggestion. MY_USART is used for both MG12 and MG24.